test/models/validatable_test.rb

```diff
@@ -1,10 +1,9 @@
 # encoding: UTF-8
 # frozen_string_literal: true
-
-require 'test_helper'
+require "test_helper"
 
 class ValidatableTest < ActiveSupport::TestCase
-  test 'should require email to be set' do
+  test 'should require email to be set' do      
```

Comment:
```suggestion
  test 'should require email to be set' do
```
remove redundant space